src/lib/resource.svelte.ts

```diff
@@ -7,6 +18,11 @@
  *
  * The value of the state can be accessed via the `value` property.
  *
+ * The default value of {@link options.onSuccess} is the identity function
+ * (**Note:** This might cause errors when {@link TPromise} and {@link TValue} are not they same).
```

Comment:
```suggestion
 * (**Note:** This might cause errors when *TPromise* and *TValue* are not they same).
```

TPromise and TValue are not defined anywhere so it is not possible to create a link to description or so.

Reply:
They are. In the resource function. Are you sure that this is not WebStorm not working correctly with tsdoc?

Reply:
This is also possible. I tested it again in VS Code and the it jumps to the generics when I click on the "TPromise" link for example. So probably it is WebStorm, although i am still the opinion that it shouldn't be a link as they are generic values, but we can keep it.